contrib/opencensus-ext-dbapi/setup.py

```diff
@@ -39,7 +39,7 @@
     include_package_data=True,
     long_description=open('README.rst').read(),
     install_requires=[
-        'opencensus >= 0.5.dev0, < 1.0.0',
+        'opencensus >= 0.3.0, < 1.0.0',
```

Comment:
0.5?

Reply:
It actually doesn't need the newer version since nothing that it uses has changed since `0.3.0`. I changed the requirement to `0.5.dev0` on master for this and all other ext packages so it would be impossible to install a non-development version in development.

Reply:
They'll still install a newer version via `pip`, this is just the minimum required version.

Reply:
I'm willing to always require the most recent version instead. It costs us some flexibility, but would make releases simpler.

Reply:
Got it, up to you :)

My personal preference would tend towards simplicity than flexibility (e.g. technically we can run some .NET 4 libraries on .NET 2, but why would we want to encourage that in the first place?).

I think the major outcome of having individual ext packages are:
1. People doesn't have to install everything if they don't need them.
2. It makes ext update (e.g. if there is a MySQL major version change which we want to support) easier without having to release the entire opencensus.

Using the ext package with an old version of opencensus wasn't our original goal, although one can argue there might be some benefits.

Reply:
I agree the benefit isn't worth the maintenance cost here, changed all of these in e4201f7.